src/screens/ConfigurePMTs/PaymentMethodConfigTypes.res

```diff
@@ -0,0 +1,24 @@
+type paymentMethodConfiguration = {
+  payment_method_index: int,
+  payment_method_types_index: int,
+  merchant_connector_id: string,
+  connector_name: string,
+  profile_id: string,
+  payment_method: string,
+  payment_method_type: string,
+  card_networks: array<string>,
+  accepted_currencies: option<ConnectorTypes.advancedConfigurationList>,
+  accepted_countries: option<ConnectorTypes.advancedConfigurationList>,
+  minimum_amount: option<int>,
+  maximum_amount: option<int>,
+  recurring_enabled: option<bool>,
+  installment_payment_enabled: option<bool>,
+  payment_experience: option<string>,
+}
```

Comment:
Are all these values we are sure gonna come?

If it's an option we can define in this way for better code readability.

```
type paymentMethodConfiguration = {
  payment_method_index: int,
  payment_method_types_index: int,
  merchant_connector_id: string,
  connector_name: string,
  profile_id: string,
  payment_method: string,
  payment_method_type: string,
  card_networks: array<string>,
  accepted_currencies?:ConnectorTypes.advancedConfigurationList,
  accepted_countries?: ConnectorTypes.advancedConfigurationList,
  minimum_amount?: int,
  maximum_amount?: int,
  recurring_enabled?: bool,
  installment_payment_enabled?: bool,
  payment_experience?: string,
}
```

The result of these two probably will be same won't cause any issue check once. If we want keys to be there then we can go with the coded approach else this

Reply:
check done based on some and none so this cannot be implemented